### PR TITLE
Create information-gathering.sh

### PR DESCRIPTION
**Pull request by arthurwolf**
This is the script for the ticket WPB-2571

### CODE REVIEW

File: bin/information-gathering.sh [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+save_file 63    crontab            /etc/crontab
+
+# Save Cron files
+save_command 64 cron-files         ls -l /etc/cron.d/*
+
+# Save Cron contents
+save_command 65 cron-contents      cat -n /etc/cron.d/*
+
+# Save DNSMasq configuration
+save_file 66    dnsmasq-conf       /etc/dnsmasq.conf
```
Comment (julialongtin) [inline bin/information-gathering.sh:208]: our customers should never have a dnsmasq.conf.

File: bin/information-gathering.sh [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+if [ "$EUID" -ne 0 ]; then
+  # If not, re-run the script with sudo
+  sudo "$0" "$@"
+  # Exit the original script, or we'll run it twice.
+  exit
+fi
+
+# Now we are running as sudo.
+
+# Installing the required packages.
```
[the remark this answers is not in the document]
Comment (arthurwolf) [reply bin/information-gathering.sh:23]: removed all sudos below this point.

File: bin/information-gathering.sh [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+# Inxi full system information 
+save_command 74 inxi-full          sudo inxi -Fxz
+
+# Inxi hardware information 
+save_command 75 inxi-hardware      sudo inxi -xxx
+
+# Detect if we are running inside a virtual machine.
+save_command 76 virt-what          sudo virt-what 
+
+# Download an Ubuntu ISO so we can see the network speed.
```
[the remark this answers is not in the document]
Comment (arthurwolf) [reply bin/information-gathering.sh:237]: changed to a smaller ( 50m ) size ISO.

File: bin/information-gathering.sh [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+save_command 44 current-language   set | egrep '^(LANG|LC_)'
+
+# Save network information
+save_command 45 network-info       ifconfig -a
+
+# Save IP addresses
+save_command 46 ip-addresses       ip addr
+
+# Save network interfaces
+save_file    47 network-interfaces /etc/network/interfaces
```
[the remark this answers is not in the document]
Comment (arthurwolf) [reply bin/information-gathering.sh]: changed to `netplan`

File: bin/information-gathering.sh [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+save_command 52 current-user       id
+
+# Save current date
+save_command 53 current-date       date
+
+# Save current UTC date
+save_command 54 current-utc-date   date --utc
+
+# Save routing tables
+save_command 55 routing-tables     ip route
```
[the remark this answers is not in the document]
Comment (arthurwolf) [reply bin/information-gathering.sh]: changed to `iptables-save`

File: bin/information-gathering.sh (hunk 1 above)
Comment (arthurwolf) [reply bin/information-gathering.sh]: removed the step, shifted all numbers.

**Pull request merged**